resources/js/components/catalog-new/ArtworksMasonry.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <div>
+        <div v-masonry transition-duration="0" item-selector=".item">
+            <div
+                v-masonry-tile
+                class="item tw-w-1/3 tw-p-2"
+                v-for="artwork in artworks"
+                :key="artwork.id"
+            >
+                <ArtworkImage
+                    :src="`${url}${artwork.id}`"
+                    :ratio="artwork.content.image_ratio"
+                    :hsl="artwork.content.hsl"
+                ></ArtworkImage>
+                <span>{{ artwork.content.title }}</span>
+            </div>
+        </div>
+        <div>
+            <button :class="{ 'tw-invisible': this.page }" ref="more" @click="$emit('loadmore')">
```

Comment:
```suggestion
            <button v-if="this.page" ref="loadMoreButton" @click="$emit('loadmore')">
```

Reply:
Not gonna work, the element is needed for IntersectionObserver:
However I can do something like this, If you prefer:
```suggestion
            <div ref="loadMoreWrapper"> 
                <button v-if="this.page" ref="loadMoreButton" @click="$emit('loadmore')">
           </div>
```

Reply:
Ah, gotcha. Would it work with `v-show`? If not, then let's out the observer on an element that's not designed to hide, like something with zero-height.

Reply:
not working with `v-show` will use the second option